UIKitDZ/UIKitDZ/LoginViewController.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  ViewController.swift
+//  UIKitDZ
+//
+//  Created by Эл on 21.09.2022.
+//
+
+import UIKit
+/// This class is for login settings
+final class LoginViewController: UIViewController {
+    
+    // MARK: - IBOutlet
+
+    @IBOutlet private weak var loginTextField: UITextField!
+    @IBOutlet private weak var emailTextField: UITextField!
+    @IBOutlet private weak var passwordTextField: UITextField!
+    @IBOutlet private weak var signinButton: UIButton!
+    @IBOutlet private weak var loginButton: UIButton!
+    
+    // MARK: - Private properties
+    
+    private var usersArray: [UserData] = []
```

Comment:
```suggestion
    private var users: [UserData] = []
```
s в конце уже понятно что множественное число значит массив